src/v/wasm/wasi.cc

```diff
@@ -116,15 +115,15 @@ uint32_t log_writer::flush() {
     if (joined.size() > max_log_line) {
         joined.resize(max_log_line);
     }
-    _logger->log(level, "{} - {}", _name, joined);
+    _logger->log(level, joined);
```

Comment:
I think this is OK for now, but we'll want to move the line buffering down into the logger at some point (maybe now is the right time?).

This has the annoying thing right now that if a guest in the VM does something like:

```
print("foo")
println("bar")
```

We log two lines:

`foo` and `bar`, when we should probably print `foobar`. This is usually seen in crashes for golang and rust, as they don't allocate memory so they just in small chunks. Anyways we don't have to fix this in this PR, but I think we should fix it as part of this project.

Reply:
gonna hold off on this to keep this PR relatively non-invasive. though I agree, part of this project, sooner rather than later.